storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -57,57 +54,34 @@ Redo_Log_Reader::Redo_Log_Reader() {
 }
 
 bool Redo_Log_Reader::find_start_checkpoint_lsn() {
-  if (log_sys->m_files.ctx().m_files_ruleset == Log_files_ruleset::CURRENT) {
```

Comment:
Can you add an assert here so that a new enum type (Log_files_ruleset) added will cause a crash or compilation failure?

Reply:
Done